### PR TITLE
fadeIn() now uses opacity to figure out the final opacity to fade into

### PR DESCRIPTION
**Pull request by niceEli**
Changed fadeIn to use opacity so it can fade into 0.5 for example
Changed Contributing.md because there was "" around linebreak style and that isnt correct
Added fadeIn example so you can learn fadeIn
Changed JSDoc to have a fadeIn example

### CODE REVIEW

File: examples/fadeIn.js
Comment (lajbel) [inline examples/fadeIn.js:1]: I am not convinced by the example. I feel it explains basic things and does not focus on fadeIn.

File: src/types.ts
Comment (lajbel) [inline src/types.ts:1]: I feel that all components are overexplained again, you have to get to the point.

**Review by lajbel — CHANGES_REQUESTED** [review]

File: examples/fadeIn.js
Comment (niceEli) [reply examples/fadeIn.js:1]: fixed. now updating PR

File: src/types.ts
Comment (niceEli) [reply src/types.ts:1]: fixed. now updating PR

**Comment by niceEli** [comment]
no clue how to rebase so ill close and make a new PR

**Comment by niceEli** [comment]
nvm turns out it auto rebases ig

**Review by lajbel — APPROVED** [review]
Is looking good

**Comment by reimertz** [comment]
this would be a welcome addition! Was just trying to figure out how to do this.

**Comment by lajbel** [comment]
can you add a changelog for this before merge?

**Comment by reimertz** [comment]
@lajbel you mean me? I can fork this and create a new PR if you want.

Should add changes under version `v3000.2.1`? if So

```
## v3000.2.1
- fixed `fadeIn()` not properly using `opacity()`
```

Also, would this be considered a breaking change since all users who specify `opacity()`, then `fadeIn()` now will see the opacity actually being applied.

**Comment by lajbel** [comment]
I was referring to the PR owner, but thank you! Good point

**Pull request merged**